pipelines/DECam/Merian.yaml

```diff
@@ -0,0 +1,195 @@
+description: |
+  The DRP pipeline specialized for the DECam instrument, developed against the
+  Merian dataset.
```

Comment:
I suggest calling this `DRP.yaml` unless there is a strong reason not to. If I am a naive new pipelines user looking for the correct DRP pipeline to run on my DECam data, I will not expect something called `Merian.yaml` to be it.

Reply:
The idea was that we _would_ have an `ingredients/DECam/DRP.yaml` someday, and this and other more concrete, specialized pipelines would mostly defer to that.  I wanted to get the directly-runnable pipelines in place before doing that refactoring on another ticket.  And I didn't want to have `pipelines/*/DRP.yaml` files because I only wanted to put things in `pipelines` that had actually been run on some dataset (and ideally either run regularly or run in some important production).  I'd rather have somebody who wants to run on new data start from the ingredients and think about what kind of processing they want, even if they end up _just_ inheriting from the ingredients pipeline.

But I think you're right that it really doesn't make sense to have just a single more-concrete pipeline file in this package that's mostly (and maybe wholly) full of quite general stuff.  And it's easy to do a lot better, by renaming this file to `ingredients/DECam/DRP.yaml` and making a new `pipelines/DECam/Merian.yaml` that just points to that - that's what I already did with the LSST/DC2 pipelines, after all.  I'll do at least that much.

I'm willing to consider going further, by also adding a `pipelines/DECam/DRP.yaml` that also just forwards to the one in ingredients; I think that's a win _if_ we end up having a single generic pipeline that can actually just work on a lot of random data.  I'm hesitant because this hasn't turned out to be the case for HSC, where our "best" pipeline is really quite RC2-specific, because the FGCM configuration is really quite dependent on the set of bands being processed together (and maybe some other properties of the dataset, but at least that).  I suppose this DECam pipeline doesn't _obviously_ have that problem, though, so maybe we should have a generic DRP.yaml until we have a concrete reason we shouldn't.  :man_shrugging:

Reply:
It's probably not worth doing on this ticket, but a future refactoring should absolutely pull out those RC2-specific FGCM configs, so that the generic parts of RC2 (lots of faro, looks like?) can be moved to a more generic place.

Reply:
I think making something that's obviously DECam-flavored `DRP.yaml` in either ingredients or pipelines is sufficient for now. I'd love the same for all cameras, just to have some vague idea of what a functional/basic DRP pipeline is for each camera. It's the kind of thing I envision new team members and folks on community asking for when they "just want to try processing some data."

Reply:
I agree with @mrawls that it would be good to have a generic-ish pipeline that new people can start from, even if it's not perfect. I'd also suggest calling this file something like `DRP-Merian`, just to make it clear that it is a specialization of DRP rather than an auxiliary pipeline (see my comment below about `RC2.yaml`).